pyaedt/application/Design.py

```diff
@@ -3152,6 +3153,17 @@ def get_evaluated_value(self, variable_name, variation=None, units=None):
         >>> M3D["p3"] = "P1 * p2"
         >>> eval_p3 = M3D.get_evaluated_value("p3")
         """
+        if self.design_type == "Maxwell Circuit":
+            if "$" in variable_name:
+                val_units = self._oproject.GetVariableValue(variable_name)
+            else:
+                val_units = self._odesign.GetVariableValue(variable_name)
+            val, units = decompose_variable_value(val_units)
+            try:
+                float(val)
```

Comment:
@maxcapodi78 I am not sure it is mandatory as we are returning the expression the line after.
```suggestion
```

Reply:
not sure either.... I'll check better